tutor/src/screens/scores-report/ux.js

```diff
@@ -108,8 +108,7 @@ export default class ScoresReportUX {
     return mapValues(averages, (average, key) => {
       const type = scoreKeyToType(key);
       let nullValue;
-
-      if (average === null) {
+      if (isNil(average)) {
         if (type === 'course_average') {
           nullValue = this.nullAverageForCourse;
         } else {
```

Comment:
average was `undefined` not `null` which are different. But `isNil` catches both cases